tests/pr_test.go

```diff
@@ -85,7 +85,24 @@ func TestRunBankingSolutions(t *testing.T) {
 			},
 		})
 
-		output, err := options.RunTest()
+		// Ignore updates
+		options.IgnoreUpdates = testhelper.Exemptions{
+			List: []string{
```

Comment:
Whats the reasoning? Is there a bug, or its expected? Perhaps a code comment would help